asciidoctorj-core/src/main/java/org/asciidoctor/cli/AsciidoctorInvoker.java

```diff
@@ -31,19 +32,13 @@ public void invoke(String... parameters) {
             jCommander.usage();
         } else {
 
-            Asciidoctor asciidoctor = null;
-            
-            asciidoctor = buildAsciidoctorJInstance(asciidoctorCliOptions);
+            Asciidoctor asciidoctor = buildAsciidoctorJInstance(asciidoctorCliOptions);
             
             if (asciidoctorCliOptions.isVersion()) {
                 System.out.println("Asciidoctor " + asciidoctor.asciidoctorVersion() + " [http://asciidoctor.org]");
                 return;
             }
 
-            if (asciidoctorCliOptions.isVerbose()) {
-                JRubyRuntimeContext.get().evalScriptlet("$VERBOSE=true");
```

Comment:
I removed this again, as it was apparently handled in the same way below in `setVerboseLevel()` which I oversaw.